cmd/opentelemetry-collector/app/defaults/default_config.go

```diff
@@ -31,37 +32,42 @@ import (
 
 // Config creates default configuration.
 // It enables default Jaeger receivers, processors and exporters.
-func Config(storageType string, factories config.Factories) (*configmodels.Config, error) {
+func Config(storageType string, zipkinPort string, factories config.Factories) (*configmodels.Config, error) {
```

Comment:
Would it be worth having an interface here to support passing in other config values if/when other flag values are used?